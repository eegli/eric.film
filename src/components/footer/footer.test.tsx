import { shallow } from '@/src/tests/testUtils';
import Footer from './footer.component';

describe('About', () => {
  it('renders card component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
