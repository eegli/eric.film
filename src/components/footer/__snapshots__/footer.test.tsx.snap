// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders card component 1`] = `
<footerstyles__FooterWrapper>
  <footerstyles__Hr />
  <footerstyles__ContentContainer>
    <footerstyles__LinkContainer>
      <Link
        href="/disclaimer#privacy"
      >
        <a>
          privacy
        </a>
      </Link>
      <Link
        href="/disclaimer#acknowledgements"
      >
        <a>
          acknowledgements
        </a>
      </Link>
      <Link
        href="/disclaimer#imprint"
      >
        <a>
          imprint
        </a>
      </Link>
    </footerstyles__LinkContainer>
    <footerstyles__GitHubIcon
      href="https://github.com/eegli/eric.film"
      rel="noopener noreferrer nofollow"
      target="_blank"
    >
      <FaGithub />
    </footerstyles__GitHubIcon>
  </footerstyles__ContentContainer>
</footerstyles__FooterWrapper>
`;
